PCbuild/prepare_ssl.py

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env python3
+# -*- coding: utf8 -*-      for <2.7>/PC/VS9.0/build_ssl.py
```

Comment:
The this is recommended form of the encoding declaration per:
  https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations
unless you are referring to the trailing comment.

The trailing comment is there to keep the line from being deleted again.